### ISSUE TITLE
working on opentaiko v0.5x

### ISSUE THREAD

**Issue by Isiah-maker** [MoshirMoshir/Moshir-OpenTaiko-Skin#3]
i want this cool skin to be compatable with Opentaiko v5.0.x PLEASE!

**Comment by MoshirMoshir** [comment]
Oh! Really? Sure! I was waiting for the official 0.6.0 release before continuing to work on this skin but I can start adding all the new assets now and port it later.

I'll see if I can make a compatible version for v0.5.4 :)

**Comment by Isiah-maker** [comment]
OMG! also make it work with all open taiko v0.5 versions not just v0.5.4.

**Comment by MoshirMoshir** [comment]
Yeah np, basically it seems for the most part I just need to rescale everything to fit 720p. It's easy but a bit tedious since I can't just rescale everything at once since there are also a few changes here and there. I started to work on it in the v0.5.4 branch, it'll take a hot minute but I should finish by the end of the month.

**Comment by Isiah-maker** [comment]
thank's bro.
